libs/traits/src/lib.rs

```diff
@@ -544,7 +572,7 @@ pub trait TokenSwaps<Account> {
 	fn cancel_order(order: Self::OrderId) -> DispatchResult;
 
 	/// Retrieve the details of the order if it exists.
-	fn get_order_details(order: Self::OrderId) -> Option<Self::OrderDetails>;
+	fn get_order_details(order: Self::OrderId) -> Option<Swap<Self::BalanceOut, Self::CurrencyId>>;
```

Comment:
@wischli. I needed to move `Swap` to the interface section, because now it's used in the `Swaps` trait. For that, I use the `Swap` type directly here. But I suppose this will be refactorized by you with a new type. Sorry for the conflicts on that! 🙏🏻

Reply:
I guess, previously, it was generic because `Swap` was in `cfg-types` and `cfg-traits` has no access to it. But it is expected to always have that value so I would say yes.